### DISCUSSION TITLE
Hm...

### DISCUSSION THREAD

**Discussion by UkrainianBanderasCat** [UkrainianBanderasCat/Digital-Logic-Sim2#60] [Announcements]
![Screenshot from 2022-07-05 10-30-40](https://user-images.githubusercontent.com/48175036/177274146-9cc31c9f-31a3-4706-b979-47b2c4bea489.png)
![Screenshot from 2022-07-05 10-31-45](https://user-images.githubusercontent.com/48175036/177274217-cd03ea26-c974-45ce-a032-b25840192c4a.png)

**Comment by UkrainianBanderasCat** [comment]
![Screenshot from 2022-07-05 10-32-56](https://user-images.githubusercontent.com/48175036/177274482-95bf75bd-5be2-40e7-8462-7a3542749c4c.png)

**Comment by firecerne** [comment]
nice

**Comment by UkrainianBanderasCat** [comment]
![Screenshot from 2022-07-06 21-01-20](https://user-images.githubusercontent.com/48175036/177614175-c71d3e77-a5f6-4dad-b9af-18eb7e0de15f.png)

**Reply by firecerne** [reply]
It is good to know people are interested in the project !
We should probably fix the problems caused by the "Pin on either side" feature as soon as possible, to be able to build the project because there is still the unusable pre-rewrite import feature, and the copy and paste feature would also be great.
To fix this problem, we could try to revert the project to when your feature was not in the repository yet.
You could just make a branch and put it there until the feature is fully functional.
It would allow us to make and test other, most wanted changes such as the Edit Chips system or a bigger board.

**Reply by UkrainianBanderasCat** [reply]
I think it is very simple to fix

**Comment by firecerne** [comment]
Can we have a little update on the numbers ?

**Reply by UkrainianBanderasCat** [reply]
![Screenshot from 2022-07-19 17-56-27](https://user-images.githubusercontent.com/48175036/179781947-b998f132-f5dd-4090-8431-cc8d4137d1c9.png)
![Screenshot from 2022-07-19 17-56-18](https://user-images.githubusercontent.com/48175036/179781957-a8f9a9ff-8737-4752-ac83-d99b063decf0.png)

**Reply by firecerne** [reply]
Maybe we could attract more players with smaller but more frequent updates, which could be made like you said in your latest announcment

**Reply by UkrainianBanderasCat** [reply]
Yeah

**Reply by firecerne** [reply]
What will the money be used for ?

**Reply by UkrainianBanderasCat** [reply]
I don't know, we can use them for advertisement or we can use them to share game on platforms like Steam, or Microsoft store(if I good remember)

**Reply by UkrainianBanderasCat** [reply]
Go to https://partner.steamgames.com/ in a web browser. This is the website for Steamworks. Here you can sign up to be a game developer. You'll need to fill out some paperwork in order to sign up for Steamworks. Before you proceed, you should have your bank account and routing number available, as well as your tax information. There is also a product submission fee of $100.00 per game you want to distribute on Steam. You will need to go through an approval process, which will take about a week.

**Reply by UkrainianBanderasCat** [reply]
I founded this in internet

**Reply by UkrainianBanderasCat** [reply]
Or even google play, we can build a game for androids. There is a little once-paid fee in 25$
But we need to have more money, so we can use just an itch

**Reply by AlphaCodingPilot** [reply]
Is the money from donations?

**Reply by UkrainianBanderasCat** [reply]
Only 17$
